compiler/generate_build_properties.gradle

```diff
@@ -1,7 +1,7 @@
 def generatedDirPath = 'generated/sources/build-properties/kotlin/main'
 
 sourceSets {
-  main.java.srcDirs += "$buildDir/$generatedDirPath"
+  test.java.srcDirs += "$buildDir/$generatedDirPath"
```

Comment:
This generates the boolean flag `USE_IR`. I only need in tests and not the main source code anymore. We still run all tests with and without the new IR backend.